composer.json

```diff
@@ -25,7 +25,8 @@
         "sebastian/diff": "~1.1"
     },
     "require-dev": {
-        "satooshi/php-coveralls": "0.7.*@dev"
+        "phpunit/phpunit": "^4.5|^5",
```

Comment:
1. tool could be a dependency
2. to develop a project you should have a phpunit. you may have global one - cool, but we don't promise that our project is compatible with it.
3. theoretically, but such an edge case...
4. it;s better to say "this one will work, you may use it" than "some phpunit could work, we dont know, try, and if it fails - well... we dont know which version works"